### ISSUE TITLE
Implement solution for linking limits

### ISSUE THREAD

**Issue by Miyayes** [brave/brave-browser#18572]
## Description

Refer to @Miyayes for spec. 

## Notes

In the meantime, support team may be able to assist you. Please open a thread on https://community.brave.com, our official support forum.

**Comment by DevloperHS** [comment]
> ## Description
> Refer to @Miyayes for spec.
> 
> ## Notes
> In the meantime, the support team may be able to assist you. Please open a thread on https://community.brave.com, our official support forum.

In how many days are we gonna see the tool, It's been 3 months. Nor there was a response after filling the wallet unlinking response. Not this type of service expected from @brave-browser-releases

**Comment by bsclifton** [comment]
Maybe @Miyayes can give an update (and close this issue)

From what I understand, we will be making a change for this soon which should eliminate the need for that tool 🙂

**Comment by forreggbor** [comment]
> Maybe @Miyayes can give an update (and close this issue)
> 
> From what I understand, we will be making a change for this soon which should eliminate the need for that tool slightly_smiling_face

That would be very welcome! Currently the 4 device limitation is a nightmare and makes the Brave rewards unuseable.

**Comment by aman-m1** [comment]
@Miyayes Let's do this.

**Comment by jarednthomas** [comment]
What a mess. I setup my dual boot laptop with Brave on each OS (Windows and Linux). Brave has been my main browser for over a year now. I also run brave on my desktop, and then my phone. Boom, 4 device limit reached. 

Now, I got a work laptop and I enjoy the built in ad blocker for brave. Yet I can't turn my "attention" into BAT anymore because of some trivial device limit that I can't control? And the expected solution is for me to go to your damned forum? Not going to happen. I hate to say it, but this is enough for me to give up on the browser altogether. Without the added benefit of BAT, I can just as easily go back to Firefox or Chrome with uBlock Origin. This is a really annoying issue and to see that this has been open for months doesn't inspire confidence.

**Comment by erintesden** [comment]
@Miyayes @bsclifton is there any topic already about that new alternative?

**Comment by bsclifton** [comment]
Hi there folks! Wanted to provide an update here. Development has started and things are moving nicely. We should have a completed change in our staging environment by late Q1 - and then available to users after that.

In the meantime, you can tag @brave/rewards-support here on GitHub or create a post on community.brave.com

**Comment by kevwil** [comment]
With a (realistically) Q2 timeframe, what happens to the rewards sitting vulnerably in the not-backed-up browser wallet and not in a "real" wallet? Could the rewards earned since the last wallet sync be lost?

**Comment by Miyayes** [comment]
> With a (realistically) Q2 timeframe, what happens to the rewards sitting vulnerably in the not-backed-up browser wallet and not in a "real" wallet? Could the rewards earned since the last wallet sync be lost?

You can take a snapshot/backup by copying your `/brave-browser` folder, and copy-pasting that into new installations of Brave. You can find the path to this folder at brave://version

**Comment by Miyayes** [comment]
Hi @Saoiray:

1. It will also apply to Android. The changes are on the back-end, so they will affect all platforms equally. That said, I added OS/Android label in any case.
2. We are removing linking limits.

This change is currently code-complete. However, actual release date (which won't be tied to a Brave release #, since it's back-end work), will depend on how testing goes, which is starting up now.

**Comment by rexsllemel** [comment]
> Hi @Saoiray:
> 
> 1. It will also apply to Android. The changes are on the back-end, so they will affect all platforms equally. That said, I added OS/Android label in any case.
> 2. We are removing linking limits.
> 
> This change is currently code-complete. However, actual release date (which won't be tied to a Brave release #, since it's back-end work), will depend on how testing goes, which is starting up now.
> 
> I renamed this issue to something more general, because we are no longer providing an interface to manage linking slots, given that linking limits will not exist anymore. I am leaving it here even though most work is happening on the back-end, just to keep visibility on this issue.

Thank God! Finally, the wait is over <3

**Comment by MuffyOne** [comment]
Any news on this? I see it's under test for more than 20 days.

**Comment by Saoiray** [comment]
@bsclifton and @Miyayes 

Frustrating, as emails and all were sent to people saying this would go into effect in March. Been 28 says since it was "code complete" and sent to testing. No updates anywhere. Would greatly appreciate if anyone can advise as to the status of implementation. 

Thanks again for your time. Hopefully can see this done ASAP as we have lots of people experiencing issues with the Device Limit.

**Comment by forreggbor** [comment]
> @bsclifton and @Miyayes
> 
> Frustrating, as emails and all were sent to people saying this would go into effect in March. Been 28 says since it was "code complete" and sent to testing. No updates anywhere. Would greatly appreciate if anyone can advise as to the status of implementation.
> 
> Thanks again for your time. Hopefully can see this done ASAP as we have lots of people experiencing issues with the Device Limit.

I would say almost everybody has the device limit problem who use Brave rewards. I couldn't count the numbers but I suppose 100 thousands people might be affected by this really stupid and meaningless limitation.

I remember when this kind of discussion started here and on Brave's support forum. The first reaction was that the limitation is set by Uphold. Even Brave's developers doesn't really understand the situation.
There was a guy on Brave's forum who unambiguously said that this is Uphold's fault. Later I realized that that guy might be a paid troll because s/he commented almost everything in every single topic and every comment was about that Brave is the god and everybody should go to hell who says the opposite.
I had a long discussion with Uphold and they confirmed that there is no limit on their side. There was others as well who talked to Uphold and confirmed my findings. The troll guy tried to disprove this statement so I left Brave's useless support forum because this troll guy came to my face every time.

**Comment by elcoinkeur** [comment]
It's even more of a problem, since the manual unlinking done using the unlinking form never worked for me. The last time I tried, it only allowed me to add one device, that's it. How can removing a limit be that hard to remove, and test ?

**Comment by rexsllemel** [comment]
> @bsclifton and @Miyayes
> 
> Frustrating, as emails and all were sent to people saying this would go into effect in March. Been 28 says since it was "code complete" and sent to testing. No updates anywhere. Would greatly appreciate if anyone can advise as to the status of implementation.
> 
> Thanks again for your time. Hopefully can see this done ASAP as we have lots of people experiencing issues with the Device Limit.

It doesn't work that way. If the code is complete and went to testing then it should probably BETA. You can't force it to release. If they release it  and has flaws, then that should be security problem and all of us will suffer. We have to wait cuz that's not magic. smh

**Comment by fearlessforhim** [comment]
> It doesn't work that way.

Seconding this. Software development isn't magic.
Yes the device limit is a pain, but more than likely there was a purpose for it at the beginning that must have seemed reasonable at the time. Now they are removing it so let's be thankful for that. Let's also recognize that MILLIONS of people use Brave linked with Uphold. Rushing out a change that is not fully tested/vetted could cause a lot (A LOT) of pain.

**Comment by filipworksdev** [comment]
I subscribed to this thread because when this issue is closed means the change is in effect. Sadly all I get is email spam in my inbox from you guys.  Now I have to unsubscribe and resume my manual checking weekly to stop the email spam.  Listen I get it I have so much BAT trapped in my computer after installing Windows 10 a few times plus my phone plus having to revert that awful Windows 11 which made me reach 4 devices. They already announced the change and and it will happen.

**Comment by Saoiray** [comment]
I tagged staff and only need staff responding. I will quickly respond to those who said something to me. You went off about how development isn't magic and all. But I didn't even say that. What I asked for was an update on the status. "We've been having issues, no ETA on arrival." or "We had some delays but it's coming, ideally available on April ##"  Lots of things that can be said. This is especially kind of critical when people shared emails from Brave that showed that the removal of the limits would be effective in March, yet we're in April without anything being said.

Now do them and everyone else a favor, please leave it be for direct contact relating to this issue. If you have something to say to me directly, you can get me over on Brave Community, I'm Saoiray there. Let's try not to clutter things here if you could help with that. 

Thanks.

**Comment by spa31** [comment]
Hello,

Could it be possible to have a ETA on this topic?

I have my wallet for years now because I had to reinstall my PC and the manual unblocking by ticked does not worked for me.

I can't understand how we are supposed to help with donations if our wallets are blocked 🤔

Thanks for your time,

**Comment by ismailozdemirx** [comment]
This uncertainty is really starting to bother me. You said March, it didn't happen. You said the beginning of April, we are in the middle of it now. Can someone in authority explain?

**Comment by Miyayes** [comment]
Hi all, thanks for your comments. I can state that the linking limit removal is live in our testing environment, but has not yet been deployed to the live production environment. We are still in the midst of testing. When we find issues during testing (which we have), we have to roll code changes to resolve those issues. Sometimes these fixes require larger changes than expected, which adds to the difficulty of giving an exact release date. 

I also want to note that the timeline was affected as we pivoted from a solution that gave users the ability to manage linking limits (which was essentially code-complete), to one that removed linking limits altogether. We pivoted because although it would take longer to implement, and code we wrote would end up unused, we knew it would be the better path in the long run. 

In any case, I regret that specific dates were mentioned, since we know it can lead to certain false expectations. 

Our current target—and it's important to say "target", not precise release date—is to get everything live in production by the end of April, as I've communicated on some other socials. However, it's possible we don't feel confident enough on exactly the last day of April. But we're getting very close to the finish line. 

_Some more context about linking limit removal:_ This particular change is big in the sense that the consequences of getting it wrong are potentially serious. **We've relied on linking limits for a long time, and they serve an extremely important purpose in our ecosystem. And we are further aware that there are bad actors who are chomping at the bits for the linking limit to be removed.** So, we want to make sure we feel confident when we decide to deploy to production. 

That said, we understand and empathize with legitimate users who've been stuck at the linking limit. The support team and I have been assisting users in the meantime trying to help them with unlinking as much as we can. It's a manual process, but I am willing to help and do it until the above work is complete. 

If you are still struggling with the linking limit, @archmeister8 @spa31 @elcoinkeur, I can personally help you. It's very important to follow these instructions so that I can effectively clear a linking slot for you ASAP without a lot of back-and-forth correspondence: 

**If you have access to any of your devices that were verified (one of your first 4 devices to verify)**, please send an email to me with:

1.  The brave://rewards-internals general info tab of that device;
2.  A list of the brave://rewards-internals general info tab of all the currently verified devices that you'd like to *keep*. I will remove a device that you do not list.

**If you do *not* have access to *any* of your devices that were verified (none of the first 4 devices)**, then:

1. If you are on Uphold, please log into your Uphold account and find your Uphold Member ID in settings inside the Uphold.com dashboard, and send that to me. I will then unlink an old devices of yours at random. 
2. If you are on Gemini, then it may not be possible, and it would be unusual for the linking limit to have been reached, as the Gemini integration was released on August 31, 2021. But you can send me an email in any case and we'll try to figure something out. 

You can send me an email with subject: `Unlinking request from Github`. My email is christopher@brave.com.

**Comment by InchaosDB** [comment]
This is inaceptable waiting for months to this solve, between you are getting the benefit of all bats we lost and of the bats we have stored just stucked because cant to send to uplhold, and promise ever at the ending of march for too late this going to was able, this need to be first priority to solve, and more ridiculous is the bats not going to attach all to same account making anyone separate browser like is totally new system. Today comes update and adivine:  no wallet fix! omg take care on users and stop that excuses.

**Comment by Vignesh-Ganesan-008** [comment]
> 1. brave://rewards-internals

I too have the same problem, can I send you an email containing the details?

**Comment by Miyayes** [comment]
Hi all, just wanted to give an update. We had a meeting yesterday to remaining parts over the finish line. Work continues. In any case, our plan is to hold on the release of linking limit removal till after May 2022 ad earnings payout (which will take us past the end of April), as we _really, really_ want the upcoming payout to go smoothly, and we do not want to confound any possible payout issues in early May with additional variables. (We are still fighting issues from the April 2022 payout, which was quite hard on everyone.) You can get some more context here: https://www.reddit.com/r/BATProject/comments/u7e8rz/april_2022_ads_payout_issues_update/.) 

That said, as promised, I've been helping dozens of users with their linking limits and helping unlink old devices for users who reached out via email using the instructions above in the meantime. I will continue to do this until linking limits are removed.

Thanks everyone for your understanding. Can't wait to get linking limits removed.

**Comment by yuceltoluyag** [comment]
I also faced this problem today. I sent you an e-mail, I would be glad if you can help. Thank you for the information @Miyayes

**Comment by 00Money00** [comment]
seems update is rolled out, i just successfully verified my wallet again with upholad after around a year.

thnx @Miyayes and whole brave team ;-)

**Comment by diegoporfirio** [comment]
> seems update is rolled out, i just successfully verified my wallet again with upholad after around a year.
> 
> thnx @Miyayes and whole brave team ;-)

Apparently nothing has changed https://brave.com/latest/

**Comment by dieterblancke** [comment]
> > seems update is rolled out, i just successfully verified my wallet again with upholad after around a year.
> > thnx @Miyayes and whole brave team ;-)
> 
> Apparently nothing has changed https://brave.com/latest/

Do remember that the change wouldn't need a Brave update, as the issue was solely on the back-end.
As mentioned before, they wanted to wait until after the April payouts, so it shouldn't have been released just yet?
I'll try myself in a bit and edit this comment afterwards.

**Comment by diegoporfirio** [comment]
> > > seems update is rolled out, i just successfully verified my wallet again with upholad after around a year.
> > > thnx @Miyayes and whole brave team ;-)
> > 
> > 
> > Apparently nothing has changed https://brave.com/latest/
> 
> Do remember that the change wouldn't need a Brave update, as the issue was solely on the back-end. As mentioned before, they wanted to wait until after the April payouts, so it shouldn't have been released just yet?
> 
> Edit: forgot I had time anyway, I'm still getting this error `Error: Device limit reached`, so I'm pretty sure it hasn't been rolled out yet. And as they are probably gonna wait until after April payouts, it can still take a good 2-3 weeks I'm guessing?

When a fix/update is released, developers close the issue in git. This one is still open.

**Comment by andybones** [comment]
> > > seems update is rolled out, i just successfully verified my wallet again with upholad after around a year.
> > > thnx @Miyayes and whole brave team ;-)
> > 
> > 
> > Apparently nothing has changed https://brave.com/latest/
> 
> Do remember that the change wouldn't need a Brave update, as the issue was solely on the back-end. As mentioned before, they wanted to wait until after the April payouts, so it shouldn't have been released just yet?
> 
> Edit: forgot I had time anyway, I'm still getting this error `Error: Device limit reached`, so I'm pretty sure it hasn't been rolled out yet. And as they are probably gonna wait until after April payouts, it can still take a good 2-3 weeks I'm guessing?

OR longer.. Apparently..
https://community.brave.com/t/brave-rewards-linking-limit/382205

**Comment by 00Money00** [comment]
> seems update is rolled out, i just successfully verified my wallet again with upholad after around a year.
> 
> thnx @Miyayes and whole brave team ;-)

seems its not fixed for everyone
donno whats happening, but among my friends, it is fixed for some and not for tohers.

maybe they are fixing accounts with limit by manual check or update is being tested on some users.

whatever it is, it solved my issue and now i have brave online everywhere i wanna use

**Comment by raycekar** [comment]
Just updated on my end. I am one that the limit still affects

**Comment by iheb2b** [comment]
> Just updated on my end. I am one that the limit still affects

same thing

**Comment by iheb2b** [comment]
any updates guys ?

**Comment by Saoiray** [comment]
@iheb2b and others, the update was already given a long time ago:

`In any case, our plan is to hold on the release of linking limit removal till after the ad earnings payout at the beginning of May 2022 (which will take us past the end of April), as we really, really want the upcoming May payout to go smoothly, and we do not want to confound any possible payout issues in early May with additional variables.`

So it's not rolled out, they are waiting until the payout that's about to begin to process. Then if all goes well, they will release it. Miyayes and/or other staff will comment here WHEN they release it. Ideally, people would stop commenting here until then so they can focus on getting things done AND do those of us tracking this issue won't keep getting spam mail from all these useless comments. 

Please do be patient everyone, it's coming. Thanks.

**Comment by InchaosDB** [comment]
Still cant link my wallets, so funny

**Comment by shayanahmad786** [comment]
Please implement asap as we are frustrated of linking limits as many of us reinstalled brave or reinstalled windows leading to loose BAT and Slots also in Mobile due to unintended uninstallation

**Comment by dylantjb** [comment]
Can someone lock this issue until it closes please?

**Comment by filipworksdev** [comment]
If you're getting spammed just go up top to notifications and do 
![image](https://user-images.githubusercontent.com/99937119/167752987-8f9e7610-c10d-4d13-901d-af9dbe25026a.png)
then
![image](https://user-images.githubusercontent.com/99937119/167753041-a2aa70bd-15b4-4831-aacf-69232dfaf8a8.png)

When the thread closes you will get a notification and you will know the issue is resolved .

**Comment by Saoiray** [comment]
@rahulbajaj369 please be sure to read what has been said here already. As has been said and later quoted in repetition:

`In any case, our plan is to hold on the release of linking limit removal till after the ad earnings payout at the beginning of May 2022 (which will take us past the end of April), as we really, really want the upcoming May payout to go smoothly, and we do not want to confound any possible payout issues in early May with additional variables.`

Since not everyone has received payout, then that means we aren't getting anything. You'll have to wait until AFTER payments are all sent for them to even have a chance to implement this. They'll update us when it's time. We should only be asking for updates or ETA when they have failed to meet timelines and haven't updated. But we're far from that. Usually payouts don't complete for 2-3 weeks. So the end of May.

**Comment by t-kauf** [comment]
> @rahulbajaj369 please be sure to read what has been said here already. As has been said and later quoted in repetition:
> 
> `In any case, our plan is to hold on the release of linking limit removal till after the ad earnings payout at the beginning of May 2022 (which will take us past the end of April), as we really, really want the upcoming May payout to go smoothly, and we do not want to confound any possible payout issues in early May with additional variables.`
> 
> Since not everyone has received payout, then that means we aren't getting anything. You'll have to wait until AFTER payments are all sent for them to even have a chance to implement this. They'll update us when it's time. We should only be asking for updates or ETA when they have failed to meet timelines and haven't updated. But we're far from that. Usually payouts don't complete for 2-3 weeks. So the end of May.
> 
> _Edit_ It was showing incomplete as I wrote this, but now seeing where it shows complete. Yet I didn't receive my payments, so I guess that's a separate issue. Also seems other people are having issues. So guess we'll see what's going on.

Browser user payouts are complete but I believe they are also waiting until after creator payouts are complete (starting 13th I believe).

As others have said, though, it's not necessarily IMMEDIATELY after payouts are complete. Many of you seem to not understand the complexity of software releases and all of the steps that you must go through.

While we wait for the fix, remember you can submit an email request to have old devices removed (info in a comment above).

**Comment by InchaosDB** [comment]
hahaha other 2-3 weeks like is aid stop lie, say the real eta time to fix, 1 year? just say dont lie more!

**Comment by betheodo** [comment]
> хахаха другие 2-3 недели, как я уже говорил, легко предсказать, никогда не бывает времени, которое вы даете, перестаньте лгать, скажите реальную дату, чтобы исправить, 1 год? просто скажи не лги больше!

Calm down, they won't do anything. They'll find some problem or reason again.
We'll still be living with four limits.
Gemini and Uphold are full of shit. We can only admire the tokens in our account.

**Comment by aman-m1** [comment]
Amazing. I'm planning to link all my Uphold linked wallets to Gemini. Can't wait.

**Comment by tallthom** [comment]
Just checking to see if that will also work with BitFlyer in Japan...

**Comment by UjCbFwtBayFM** [comment]
Will it still work with countries being removed from support lists? Gemini wiped a lot of them recently and Uphold made a similar move a few month ago.

**Comment by Miyayes** [comment]
> Just checking to see if that will also work with BitFlyer in Japan...

Linking limits still in place for bitFlyer in Japan for the moment, but we will remove them for bitFlyer too in the very near future.

**Comment by drkdw** [comment]
Impatiently waiting for todays **closed** label! 🤞

**Comment by betheodo** [comment]
> Привет всем, просто хотел дать обновление здесь. Сегодня у нас было собрание, чтобы рассмотреть все для запуска, и мы планируем запустить его где-то в эту среду (25 мая 2022 г.). Спасибо всем.

Will you let me know here when it's ready?

**Comment by tallthom** [comment]
> > Just checking to see if that will also work with BitFlyer in Japan...
> 
> Linking limits still in place for bitFlyer in Japan for the moment, but we will remove them for bitFlyer too in the very near future.

Thank you. I eagerly await that future.

**Comment by rahulbajaj369** [comment]
Anyone get the update yet?

**Comment by Saoiray** [comment]
> Anyone get the update yet?

There won't be an update. When it rolls out, it's going to be on the back-end. They haven't said anything here in terms of whether it's been done yet or what the status might be. Hopefully @Miyayes will comment here as soon as it's implemented OR if they see there's going to be a delay.

**Comment by Miyayes** [comment]
And we're live. I will update here (and on our other forums, such as https://community.brave.com) if necessary.

**Comment by Saoiray** [comment]
> And we're live. I will update here (and on our other forums, such as https://community.brave.com) if necessary.

Thanks so much for the update. Hopefully no issues on it.

**Comment by cfeller** [comment]
It was working - I was able to (finally, a year later), link Brave on my phone, then my new laptop, all good, then I rebooted into Linux to get it from the Linux side (Error: Device limit reached). 

So I was able to add a few outstanding devices (yay!), but it appears I'm running up against a wall again.

**Comment by Miyayes** [comment]
@cfeller can you send me your brave://rewards-internals of a device that you got successfully linked just now? Will check what's going on in your case! You can send it to me at christopher at brave dot com. Be sure to include your username in the email so I know which one's you :)

**Comment by alemansrao** [comment]
Im getting "Something went wrong. Please try again" on uphold when i try to link

**Comment by hiimhawkeye** [comment]
@Miyayes tried to link a single device and still being hit with an error on Uphold. Using Android device, once I input my authenticator code it sends me to Uphold app main screen with a red banner up top saying "something went wrong. Please try again", then I get a notification saying "you've just authorized brace browser" and the app says integrated. How will I know if it actually successful?

**Comment by Miyayes** [comment]
> @Miyayes tried to link a single device and still being hit with an error on Uphold. Using Android device, once I input my authenticator code it sends me to Uphold app main screen with a red banner up top saying "something went wrong. Please try again", then I get a notification saying "you've just authorized brace browser" and the app says integrated. How will I know if it actually successful?

That's a different issue. You'll need to uninstall Uphold app so that when you verify, you stay in the Brave Browser app. 

We think this has fixed itself in the new versions of Brave, but in current version, you'll need to uninstall Uphold app while you're verifying Brave Rewards.

**Comment by t-kauf** [comment]
@Miyayes Thank you for getting this out! I was able to successfully link my 5th device, however, it only transferred 9.5 out of 21.25 BAT that I had on the device to my Uphold account. Should I send details from brave://rewards-internals as well?

**Comment by Ghost4592** [comment]
@Miyayes i am living in India and when did i cant connect to gemini because gemini says region not supported but i used to get bat in my gemini acccount directly which was linked.

**Comment by Yenniferh** [comment]
Thank you guys. It's working like a charm on Linux with Uphold wallet

**Comment by auxnsubs** [comment]
reporting it works well for me, Brave browser on Windows 10, with Gemini wallet.
Thank you @Miyayes and team!

**Comment by Saoiray** [comment]
> i am living in India and when did i cant connect to gemini because gemini says region not supported but i used to get bat in my gemini acccount directly which was linked.

India is no longer supported by either.

You can also see that Uphold shows on their website that they do not support India at https://support.uphold.com/hc/en-us/articles/205845376-Where-does-Uphold-offer-bank-account-funding-and-withdrawals- as it is not one of the 35 countries where they are doing business.

As to Gemini allowing accounts, the list of accepted countries can be seen at [NOTICE: New region restrictions on verifying Brave Rewards with Gemini](https://community.brave.com/t/notice-new-region-restrictions-on-verifying-brave-rewards-with-gemini/393482) and sadly India is not listed there.

**Comment by zihaaad** [comment]
can confirm it worked finally.
Debian 11.

**Comment by filipworksdev** [comment]
Maan I was hoping is gonna last 1 more month. I loved reading the angry posts. Thanks guys. Can confirm it works.

**Comment by drkdw** [comment]
Working!

**Comment by dazzat1960** [comment]
Can also confirm that it works now.

**Comment by veja1796** [comment]
just tried to log in and link my browser but my profile is flagged. I lost my last linked device in January I don't understand why my profile flagged. 

What can I do to unflag my profile?

**Comment by betheodo** [comment]
I connected my two devices to Uphold and the balance of coins disappeared. It did not appear in Uphold.Where are my coins?

**Comment by drkdw** [comment]
@betheodo im using Gemini. But I guess it works the same  Once you connect your new device the saved coins will automatically be synced to your account.

**Comment by betheodo** [comment]
> @betheodoя использую Gemini. Но я думаю, это работает так же. Как только вы подключите новое устройство, сохраненные монеты будут автоматически синхронизированы с вашей учетной записью.

I wrote that they were missing and out of sync.

**Comment by diegoporfirio** [comment]
Working =)

**Comment by rahulbajaj369** [comment]
@Miyayes While I was able to link a browser to Gemini, I lost the BAT that was already on that browser. Has anyone else experienced the same?

**Comment by betheodo** [comment]
> @MiyayesХотя мне удалось связать браузер с Gemini, я потерял BAT, который уже был в этом браузере. Кто-нибудь еще испытал то же самое?

Yes, I connected Uphold and lost my tokens.

**Comment by drumstick77** [comment]
So I've managed to link my android device and it can only be done via
browser method not app. So if you have the app, uninstall it first.
Where I've stumbled though for some odd reason is the bowser on windows 11.

got this error. Anyone got the same error?

[image: image.png]

**Comment by Miyayes** [comment]
> @Miyayes While I was able to link a browser to Gemini, I lost the BAT that was already on that browser. Has anyone else experienced the same?

After 10 or so mins, did they appear in your Gemini account?

**Comment by Miyayes** [comment]
> So I've managed to link my android device and it can only be done via browser method not app. So if you have the app, uninstall it first. Where I've stumbled though for some odd reason is the bowser on windows 11. got this error. Anyone got the same error? [image: image.png]
> […](#)
> Message ID: ***@***.***>

Sorry, can't see your screenshot. Uphold or Gemini?

**Comment by drumstick77** [comment]
Hi Chris,

It's uphold

please see attached.
Many Thanks,
Jonathan

**Comment by betheodo** [comment]
> > @MiyayesХотя мне удалось связать браузер с Gemini, я потерял BAT, который уже был в этом браузере. Кто-нибудь еще испытал то же самое?
> 
> Примерно через 10 минут они появились в вашей учетной записи Gemini?

It's been more than 7 hours and still no tokens have appeared in my Uphold Wallet

**Comment by andybones** [comment]
working for me!!
linked phone, my work laptop and home PC!
thank you guys!

**Comment by t-kauf** [comment]
I was able to successfully link my 5th device, however, it only transferred 9.5 out of 21.25 BAT that I had on the device to my Uphold account. My browser's event logs shows "21.25 claimed". Should I send details from brave://rewards-internals as well? Or should I post in the forum?

**Comment by spa31** [comment]
To get it done on android I had to un install uphold app first so the account linking authentication goes by web and then it works.

This night I'll try it on my windows 11 to see if it goes through.

Thanks

**Comment by rahulbajaj369** [comment]
> > @Miyayes While I was able to link a browser to Gemini, I lost the BAT that was already on that browser. Has anyone else experienced the same?
> 
> After 10 or so mins, did they appear in your Gemini account?

Yep! All good here!

**Comment by filipworksdev** [comment]
So at first it looked like my BAT vanished but appeared much later anyway.

**Comment by betheodo** [comment]
> Итак, сначала казалось, что мой BAT исчез, но все равно появился гораздо позже.

Can you tell me more about it? What wallet did you connect? What was the balance? When was it credited to the wallet?

**Comment by spa31** [comment]
Working on Windows 11 too...

Thanks for the development.

**Comment by Ghost4592** [comment]
@Miyayes bro i am having the same problem device limit reached when i did through uphold

**Comment by DariuszPakulski2137** [comment]
I managed to connect all 4 devices but when I tried to add 5th it says limit reached, what if they broke and I must install Brave again, will I be closed forever? or it resets after some time?

**Comment by Day-dreamer0** [comment]
I am getting this error, while connecting my 5th device....
![Screenshot from 2022-06-12 09-42-56](https://user-images.githubusercontent.com/61547878/173214095-9f3ff390-8f6b-4eb5-82de-b8dc7646ff47.png)

**Comment by Day-dreamer0** [comment]
> I am getting this error, while connecting my 5th device.... ![Screenshot from 2022-06-12 09-42-56](https://user-images.githubusercontent.com/61547878/173214095-9f3ff390-8f6b-4eb5-82de-b8dc7646ff47.png)

It says that application not available for geolocation and my location is india and I have previously used both brave and uphold without any issue.

**Comment by ElHave247** [comment]
omg and after those 2000 lies has been released omg

**Comment by Ayern12** [comment]
## help me to handle this liking..

Pada Ahd, 10 Jul 2022, 20:28 ElHave ***@***.***> menulis:
